src/monster860/fastdmm/FastDMM.java

```diff
@@ -482,27 +482,33 @@ else if(dwheel < 0)
 		
 		while(Mouse.next()) {
 			if(Mouse.getEventButtonState()) {
+				try {
+					currPopup.setVisible(false);
+				} catch (NullPointerException e) {
+					// Doesn't exist. Oh well.
+				}
```

Comment:
Put this before the if statement
```
if(currPopup != null && !currPopup.isVisible())
    currPopup = null;
```

And inside, between the setvisible and continue call put `currPopup = null`